.gitignore

```diff
@@ -0,0 +1,25 @@
+# Python
+__pycache__/
+*.py[cod]
+*$py.class
+
+# Distribution / packaging
+*.egg-info/
+dist/
+build/
+eggs/
+*.egg
+
+# Virtual environments
+venv/
+.venv/
+.direnv/
+.env/
+ENV/
+
+# IDE
+.idea/
+*.swp
+*.swo
+.vscode/
+.fleet/
```

Comment:
IMHO these are personal choices and should be in your personal global .gitignore.

We could keep a venv reference here if it's referenced from the makefile or docs, but otherwise not.